### PR TITLE
Added commands info in README.md

### PR DESCRIPTION
**Pull request by nadongguri**
Added commands info in README.md
- lint
- pr

ISSUE=#177

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ native layer.
 - You can just run the following command to test.
   - ```./bacardi test```
 
+## Lint
```
Comment (romandev) [inline README.md:33]: I think this is unnecessary. We already have `format` command in our README.md.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ native layer.
...
+- You can just run the following command to lint.
+  - ```./bacardi lint```
+ 
+## Cherry-pick a pull request
```
Comment (romandev) [inline README.md:37]: I'm not sure if it's good to write this section here.
We might have to write a new separated documentation.

File: README.md (hunk 1 above)
Comment (yjaeseok) [reply README.md:33]: IMHO, I think we need to introduce lint command. lint and format command are different

File: README.md (hunk 2 above)
Comment (yjaeseok) [reply README.md:37]: I think so too

File: README.md (hunk 1 above)
Comment (nadongguri) [reply README.md:33]: @romandev, if you think so, why don't we discuss about ISSUE #177 ?

File: README.md (hunk 2 above)
Comment (nadongguri) [reply README.md:37]: @romandev, @yjaeseok, this pr is based on the content in the issue but I also thought of adding pr command in README.md might need to be in another file.

File: README.md (hunk 1 above)
Comment (romandev) [reply README.md:33]: @yjaeseok Yeah, they are usually each different commands. But the current implementation of `lint` command is implemented based on `format` command. So, there is no difference between two commands.

If there are some lint errors, users can run `./bacardi format` command.
So, all lint errors will be fixed. Then, why do we need to expose `./bacardi lint` command to user?

If `./bacardi format` comand doesn't cover all lint errors, it should be exposed. But now I do not think there is a good reason to expose it. IMHO, it's better to expose a small number of things to users if possible.

Comment (romandev) [reply README.md:33]: @nadongguri I left some comments in the ISSUE.

File: README.md (hunk 2 above)
Comment (yjaeseok) [reply README.md:37]: @nadongguri So i think this pr could be closed, and then we can make pr for another file.

**Comment by nadongguri** [comment]
I will prepare this pull-request in more-detail.

**Pull request closed without merging**